spec/index.html

```diff
@@ -734,9 +735,153 @@ <h3>Overview</h3>
+            :p :r _:a .
+            _:a :t :u .
+          -->
+        </pre>
+
+        <p>These are then serialized to N-Quads, concatenated and hashed resulting in
+          `6fa0b9bdb376852b5743ff39ca4cbf7ea14d34966b2828478fbf222e7c764473`.</p>
+
+        <p>Thus the generated hashes each reference just a single blank node,
+          allowing the canonicalization algorithm to use only the
+          Hash First Degree Quads algorithm.</p>
+      </aside>
+
+      <aside id="ex-shared-hashes"
+            class="example"
+            title="Shared hashes">
+        <p>This example illustrates hashing quads containing
+          blank nodes where hashing the statements mentioning
+          those blank nodes have overlapping results.</p>
+
+        <figure id="fig-shared-hashes" style="text-align:center">
+          <!-- Source for this file is at https://docs.google.com/drawings/d/1HWpz8S-PNaWi02-utF62HAMd_0b-AF_tHNSRRUQILbU/edit?usp=sharing -->
+          <object data="shared-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-shared-hashes-alt">
+            <p id="fig-shared-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-shared-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in shared hashes.<br/>
+            Image available in
+            <a href="shared-hashes.svg" title="SVG image of a graph resulting in shared hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+          
+        <pre id="ex-shared-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in shared hashes">
+          <!--
+            :p :q _:b0 .
+            :p :q _:b1 .
+            _:b0 :p _:b2 .
+            _:b1 :p _:b3 .
+            _:b2 :r _:b3 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called four times, with each blank node
+          (`_:b0`, `_:b1`, `_:b2`, and `_:b3`).
+          Running the algorithm with the reference node `_:b0` results in the
+          following quads, after replacing blank nodes:</p>
+
+        <pre data-transform="updateExample">
+          <!--
+            :p :q _:a .
+            _:a :p _:z .
+          -->
+        </pre>
+
+        <p>Which hashes to: `3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`.</p>
+
+        <p>Note that using reference node `_:b1` results in the same quads,
+          and thus results in the same hash.</p>
+
+        <p>Using the reference node `_:b2` results in the following quads:</p>
+
+        <pre data-transform="updateExample">
+          <!--
+            _:z :p _:a .
+            _:a :r _:z .
+          -->
+        </pre>
+
+        <p>Which hashes to: `3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`.</p>
```

Comment:
Some of these hash values that aren't supposed to be the same are shown as the same, so they probably just need recalculation, etc.